libraries/nanoFramework.Modbus/README.md

```diff
@@ -0,0 +1,5 @@
+# nanoFramework.Modbus
```

Comment:
Markdown 101: you need a new line after a title
```suggestion
# nanoFramework.Modbus

```

Reply:
modified, please review

Reply:
I would be honored to be able to add Modbus implementing to the nanoFramework.IoT.Device repository.

I need some time to standardize and modify the code style, and when everything is ready, I pull request

Reply:
Whenever you'll be ready, go for a PR. Happy to help you and assist you.